docs/Install.md

```diff
@@ -185,6 +185,8 @@ Troubleshooting
          - In this case, either kill them manually by hitting Ctrl+C or run `sudo pkill NF_NAME` for every NF that you have ran.
      - The manager and NFs are not running, but something crashed without freeing hugepages.
          - To fix this, please run `sudo rm -rf /mnt/huge/*` to remove all files that contain hugepage data.
+         - An alternative method is to use the dpdk provided tools (the following may have changed since DPDK v18.11): Run `cd dpdk/usertools; sudo ./dpdk-setup.sh` then select "remove hugepage mappings" from the list of options. Then run `$ONVM_HOME/scripts/install.sh` to remount hugepages.
```

Comment:
Is it really correct behavior to run `install.sh` again? Would it be better to run the `setupenvironment.sh` (or whatever it is called) instead?  
Presumably we only want them to run install once but the setup script multiple times should be fine.

Reply:
I had the same thought. As long as the hugepage path stays mounted, `setup_environment.sh` should be fine. I'll change the doc to use that script instead.